api/src/main/java/jakarta/enterprise/inject/build/compatible/spi/BeanInfo.java

```diff
@@ -122,10 +122,18 @@ public interface BeanInfo {
      */
     boolean isAlternative();
 
+    /**
+     * Returns whether this bean is a {@linkplain jakarta.enterprise.inject.Reserve reserve}.
+     *
+     * @return whether this bean is a {@linkplain jakarta.enterprise.inject.Reserve reserve}
+     */
+    boolean isReserve();
+
     /**
      * Returns the {@linkplain jakarta.annotation.Priority priority} declared on this bean, or {@code null}
      * if this bean does not declare a priority. Declaring a priority on an alternative bean makes it an enabled
-     * alternative. Similarly, declaring a priority on an interceptor makes it an enabled interceptor.
+     * alternative. Declaring a priority on a reserve bean makes it an enabled reserve. Declaring a priority
+     * on an interceptor makes it an enabled interceptor.
```

Comment:
Should the alternatives and reserves be "selected" rather than "enabled" here?

Looks like the current spec for alternatives mostly says "selected" but sometimes says "enabled" :disappointed: 

I think interceptors still get enabled.

Reply:
If an alternative/reserve is selected [for the application / for the bean archive], it also becomes enabled. Interceptors do not get selected, but they are still enabled.

Reply:
It is true though that when it comes to priority, we should speak about selecting alternatives/reserves for the application. They become enabled due to being selected.